### DISCUSSION TITLE
Polling interval?

### DISCUSSION THREAD

**Discussion by drlauridsen** [tillsteinbach/WeConnect-mqtt#248] [Q&A] [answered]
Maybe I have misunderstood how this works, but the Egolf allows 15 requests before shutting of remote access until car has been turned on.

How often is the VW server polled for data?
And does it force refresh data?

**Comment by tillsteinbach** [comment, marked as answer]
Only the server is polled, not the car. The car is only getting a wake-up when you actually control it, like starting/stopping charging or turning on AC.

**Reply by drlauridsen** [reply]
Great thanks
How often is the server polled, and can this be adjusted?

**Reply by tillsteinbach** [reply]
You can use the `—interval` option and add the interval in seconds. The lower limit is 300 seconds if I remember correctly. We don’t want the server to be bothered by too many requests.
